### PR TITLE
Fx adds

### PR DESCRIPTION
**Pull request by AnneWellings**
Made the aliens worth -200, since the humans are 200. can be changed, though. also made the house animation quicker, and added a little poof when pucks disappear in driveways.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,8 @@
 [Bb]uild/
 [Bb]uilds/
 Assets/AssetStoreTools*
+Assets/ProCamera2D
```
Comment (Bloodyaugust) [inline .gitignore]: These shouldn't be necessary, we have them further down in the gitignore. 😄

File: Assets/Core/Scripts/Behaviors/changeHouses.cs [hunk 2]
```diff
@@ -28,11 +41,36 @@ void Start()
...
+                }
+                break;
+            case animStage.Wait:
+                break;
+        }
     }
 
     void UpdateSprite(HouseChangeData changeData) {
-      Sprite newSprite;
+      //Sprite newSprite;
```
Comment (Bloodyaugust) [inline Assets/Core/Scripts/Behaviors/changeHouses.cs]: This can just be removed. 😄

**Review by Bloodyaugust — APPROVED** [review]
Just some minor stuff, go ahead and make those changes then merge-ify. 👍

**Comment by AnneWellings** [comment]
Made the requested changes.

File: .gitignore (hunk 1 above)
Comment (AnneWellings) [reply .gitignore]: Haha, sorry, I didn't notice :)

**Pull request merged**